elements/pfe-autocomplete/src/pfe-autocomplete.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Autocomplete",
+  "description": "Autocomplete provides options in a dropdown list as user types in an input box by showing result from an api call.",
+  "type": "object",
+  "tag": "pfe-autocomplete",
+  "class": "pfe-autocomplete",
+  "category": "content",
+  "properties": {
+    "slots": {
+      "title": "Slots",
+      "description": "Definition of the supported slots",
+      "type": "object",
+      "properties": {
+        "content": {
+          "title": "Content",
+          "type": "array",
+          "namedSlot": false,
+          "items": {
+            "oneOf": [
+              {
+                "$ref": "raw"
```

Comment:
Since this component will only accept an input tag as the child, should this be input instead of raw?